apps/freelan/config/freelan.cfg

```diff
@@ -779,11 +779,6 @@ ipv6_address_prefix_length=2aa1::1/8
 # Default: 2000
 #passphrase_iterations_count=2000
 
-# The salt to use when deriving the PSK from the passphrase.
-#
-# Default: freelan
-#passphrase_salt=freelan
```

Comment:
You removed the whole section for the `passphrase_salt` was that intended ?

This is not the same as `passphrase`.